airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -87,6 +87,19 @@
               }
             }
           },
+          {
+            "title": "Standard Inserts",
```

Comment:
can you: 

1. set `hidden: true` to hide this option from the UI (no one should be using it anymore)
2. in the connector, use internal staging when this option is set

Reply:
@sherifnada, when `hidden` is set to `true`, what will the UI show for a connection already set to standard loading? Will it cause a UI exception?

Reply:
actually nvm, let's just do this option for now and handle what I suggested in a follow up PR just to unblock current users

Reply:
I think `hidden` won't work for enum values actually so I was wrong. migration will be a little more complex

Reply:
Yes, hidden: true won't work for oneOf. Just tested it on UI